GeosCore/drydep_mod.F90

```diff
@@ -585,30 +581,51 @@ SUBROUTINE UPDATE_DRYDEPFREQ( Input_Opt, State_Chm, State_Diag, State_Grid, &
           ! Dry deposition frequency [1/s]
           State_Chm%DryDepFreq(I,J,D) = State_Chm%DryDepVel(I,J,NDVZ) / THIK
 
-#if !defined( MODEL_CESM )
-          ! Archive dry dep velocity for diagnostics in [cm/s]
+          ! Free pointer
```

Comment:
I added the changes in this file to avoid the seg fault in GEOS and CESM that had to do with the dry dep velocity diagnostic. Moving the diagnostics setting outside of the main loop worked to eliminate the problem. I'm not 100% satisfied with that solution, but we can settle for it until a future compiler finds an issue.

Reply:
Fine with me. And it sounds like your bug fix will eliminate this anyways.